packages/server/graphql/private/mutations/processRecurrence.ts

```diff
@@ -0,0 +1,119 @@
+import ms from 'ms'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {RRule} from 'rrule'
+import getRethink from '../../../database/rethinkDriver'
+import MeetingTeamPrompt from '../../../database/types/MeetingTeamPrompt'
+import getPg from '../../../postgres/getPg'
+import {getActiveMeetingSeriesQuery} from '../../../postgres/queries/generated/getActiveMeetingSeriesQuery'
+import {MeetingSeries} from '../../../postgres/queries/getMeetingSeriesByIds'
+import {analytics} from '../../../utils/analytics/analytics'
+import publish from '../../../utils/publish'
+import standardError from '../../../utils/standardError'
+import createTeamPrompt from '../../mutations/helpers/createTeamPrompt'
+import endTeamPrompt from '../../mutations/helpers/endTeamPrompt'
+import isStartMeetingLocked from '../../mutations/helpers/isStartMeetingLocked'
+import {IntegrationNotifier} from '../../mutations/helpers/notifications/IntegrationNotifier'
+import {MutationResolvers} from '../resolverTypes'
+
+const startRecurringTeamPrompt = async (
+  meetingSeries: MeetingSeries,
+  startTime: Date,
+  dataLoader,
+  r,
+  subOptions
+) => {
+  const {teamId, facilitatorId} = meetingSeries
+
+  // AUTH
+  const unpaidError = await isStartMeetingLocked(teamId, dataLoader)
+  if (unpaidError) return standardError(new Error(unpaidError), {userId: facilitatorId})
```

Comment:
+1 this will be called for every meeting that's about to start, and it'll hit the DB every time, do we want to check if the team is clocked only once?

Reply:
So as it turns out, `isStartMeetingLocked` actually uses the `dataLoader` here, so this should only hit the DB once per teamId.